src/libopensc/sc-ossl-compat.h

```diff
@@ -114,208 +55,11 @@ extern "C" {
 #define EVP_PKEY_get_raw_public_key(p, pu, l)   (0)
 #endif
 
+/* OpenSSL 1.1.1 has FIPS_mode function */
 #if OPENSSL_VERSION_NUMBER >= 0x30000000L
-#define EC_POINT_get_affine_coordinates_GFp     EC_POINT_get_affine_coordinates
-#define EC_POINT_set_affine_coordinates_GFp     EC_POINT_set_affine_coordinates
-
-# ifndef FIPS_mode
```

Comment:
please, keep this `ifndef`. The OpenSSL 3.0 when built with FIPS module has this macro/function too.

Reply:
FIPS_mode does not show up in include/openssl/  is 3.0.0 or 3.0.1.

But late in new code is:
```
 59 #if OPENSSL_VERSION_NUMBER >= 0x30000000L
 60 #define FIPS_mode()                             EVP_default_properties_is_fips_enabled(NULL)
 61 #endif 
```

I think this  does the same thing.

Reply:
Thanks for clarification. We have for some reason this compat macro defined in `openssl/fips.h` in el9, but it looks like it does not complain about redefinition so we are fine.